Buildings/Fluid/Dehumidifiers/Desiccant/BaseClasses/Performance.mo

```diff
@@ -0,0 +1,301 @@
+﻿within Buildings.Fluid.Dehumidifiers.Desiccant.BaseClasses;
+model Performance
+  "Model calculates the outlet condition of the process air through a desiccant dehumidifier"
+    extends Modelica.Blocks.Icons.Block;
```

Comment:
delete one empty space before `extends`. Format

Reply:
Removed the extra space.